src/package/package.js

```diff
@@ -168,6 +186,7 @@ module.exports = {
           .filter((i) => i.isFile() && regex.test(i.name))
           .map((i) => i.name);
       return readdir(instPath).concat(
+        readdir(path.join(instPath, 'packages')).map((i) => 'packages/' + i),
```

Comment:
There is a bug.
I don't have enough time to suggest all the fixes, so I apologize for the messy suggestions.

```suggestion
        readdir(path.join(instPath, 'plugins')).map((i) => 'plugins/' + i),
```

Reply:
Oh, I certainly didn't check there.

I checked the other strings, but I think the only part I need to change is where you suggested.